mypyc/codegen/emitfunc.py

```diff
@@ -488,6 +502,10 @@ def visit_load_address(self, op: LoadAddress) -> None:
         src = self.reg(op.src) if isinstance(op.src, Register) else op.src
         self.emit_line('%s = (%s)&%s;' % (dest, typ._ctype, src))
 
+    def visit_keep_alive(self, op: KeepAlive) -> None:
+        # This is a no-op.
+        pass
```

Comment:
A total unrelated comment here, I like the idea of having `KeepAlive`, but I wonder whether we should keep this analysis-only IR all the way to codegen. If more analysis-only (or mid-end only) IRs are introduced in the future, it may become reasonable to eliminate them first.

Reply:
This actually gets stripped along the way. I'll try raising an exception if we reach here.